lib/modules/datasource/pypi/index.ts

```diff
@@ -83,7 +92,7 @@ export class PypiDatasource extends Datasource {
     return input.toLowerCase().replace(regEx(/(_|\.|-)+/g), '-');
   }
 
-  private async getDependency(
+  private async getResultsViaPyPiJson(
```

Comment:
Please do this refactor in a seperate PR to be merged first.

Reply:
I went to do this refactor and it's very minor - I think can be combined here